exec_test.go

```diff
@@ -35,3 +36,42 @@ func ExampleCommandExists() {
 	// true
 	// false
 }
+
+func BenchmarkCommandExists(b *testing.B) {
+	cases := []struct {
+		Command string
+	}{
+		{"go"},
+		{"gofmt"},
+		{"bazel"},
+		{"ps"},
+	}
+	for _, bc := range cases {
+		b.Run(bc.Command, func(b *testing.B) {
+			b.ResetTimer()
+			for i := 0; i < b.N; i++ {
+				u.CommandExists(bc.Command)
+			}
+		})
+		b.Run(bc.Command+"-parallel", func(b *testing.B) {
+			b.ResetTimer()
+			b.RunParallel(func(pb *testing.PB) {
+				for pb.Next() {
+					u.CommandExists(bc.Command)
+				}
+			})
+		})
+	}
+}
+
+func BenchmarkSafeExec(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		u.SafeExec(exec.Command("sh", "echo $GOPATH; exit 1"))
+	}
+}
+
+func BenchmarkExecStandaloneOutputs(b *testing.B) {
```

Comment:
this is not a standalone benchmark, if you think it may change the perf VS the BenchmarkSafeExec, just add a testcae in BenchmarkSafeExec